src/.gitignore

```diff
@@ -12,6 +12,9 @@ lib/
 
 *_wrap.c
 
+pyparsing.py
+pynusmv/nusmv/trace/exec_/exec_.py
```

Comment:
Ignoring this file should be done in the last part of the .gitignore file, with other .py files generated by SWIG.